auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -119,6 +120,7 @@ private void populateIdpList(List<IdpConfig> providerConfigs,
             @LayoutRes int buttonLayout;
             switch (idpConfig.getProviderId()) {
                 case GoogleAuthProvider.PROVIDER_ID:
+                case PlayGamesAuthProvider.PROVIDER_ID:
```

Comment:
I think we want to have a green "Sign in with Play games" button.  I'll see if I can get us the assets.